src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
```

Comment:
I dont think we need to make "supply" and "buy" as constant as their names do not add informativeness to the code (constants are called the same as values)